src/I18n/DateFormatTrait.php

```diff
@@ -125,9 +125,7 @@ public function nice($timezone = null, $locale = null)
      * $time->i18nFormat(Time::UNIX_TIMESTAMP_FORMAT); // outputs '1398031800'
      * ```
      *
-     * If you wish to control the default format to be used for this method, you can alter
-     * the value of the static `Time::$defaultLocale` variable and set it to one of the
-     * possible formats accepted by this function.
+     * You can control the default format used by setting through `Time::setToStringFormat()`.
```

Comment:
```suggestion
     * You can control the default format used through `Time::setToStringFormat()`.
```

I think the extra words make this sentence harder to understand.

Reply:
Funny, I actually debated the same thing.